src/RapidezStatamicServiceProvider.php

```diff
@@ -55,6 +55,31 @@ public function boot()
         Alternates::register();
     }
 
+    public function bootSites() : self
+    {
+        $sites = [];
+        $stores = Rapidez::getStores();
```

Comment:
We shouldn't query anything directly from the Service Provider, as it's executed even from your console when you didn't have a database connection yet. But if we move it to a middleware it's only executed on browser requests; not when you run something in the CLI. But this generates the sites config file? Maybe a command?